app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -274,6 +296,8 @@ fun MapScreen(
     profileViewModel.getProfileById(authViewModel.currentUser.value!!.uid)
   }
 
+  var zoomLevel by remember { mutableDoubleStateOf(0.0) }
+
```

Comment:
Warning: The default value should be `MapScreen.MAP_INITIAL_ZOOM` here.

Reply:
Yep! Good catch! I changed it!